src/lib/gui/ScreenSetupModel.cpp

```diff
@@ -57,6 +57,9 @@ QVariant ScreenSetupModel::data(const QModelIndex &index, int role) const
 
   case Qt::DisplayRole:
     return screen(index).name();
+
+  default:
+    return QVariant();
   }
   return QVariant();
```

Comment:
Maybe we can get away with not logging default case here as that'd probably just be noise in this case, but the return at end of function is possibly redundant.
```suggestion

  default:
    return QVariant();
}
```

Reply:
It should be be otherwise i suspect a this function could not return a value warning so leaving it for that.